DataFormats/CTPPSDigi/interface/CTPPSPixelDigi.h

```diff
@@ -0,0 +1,69 @@
+#ifndef CTPPS_PIXEL_DIGI_H
+#define CTPPS_PIXEL_DIGI_H
+/**
+ * Persistent digi for CTPPS Pixels.
+ * Author: F.Ferro ferro@ge.infn.it
+ */
+
+#include "FWCore/Utilities/interface/Exception.h"
+
+class CTPPSPixelDigi {
+public:
+
+
+  CTPPSPixelDigi( int packed_value) : theData(packed_value) {}
+
+  CTPPSPixelDigi( int row, int col, int adc) {
+    init( row, col, adc);
+  }
+
+  CTPPSPixelDigi( int chan, int adc) {
+    std::pair<int,int> rc = channelToPixel(chan);
+    init( rc.first, rc.second, adc);
+  }
+
+  CTPPSPixelDigi() : theData(0)  {}
+
+  void init( int row, int col, int adc) ;
+
+  // Access to digi information
```

Comment:
```///``` or ```//!``` would be better for inline comments, since these can be interpreted by doxygen.